pipeline/preprocessing/ground_truth_processing/open_street/osm_geometry.py

```diff
@@ -24,13 +27,18 @@ def way(self, w: Any) -> None:
         ):  # The highway tag annotates the type of road, e.g. 'path' or 'motorway'
             try:
                 geo = GeoJSONFactory().create_linestring(w)  # Get the road linestring
-                self.geometryDictionary[w.id] = (geo, w.tags.get("name"))
+                self.geometryDictionary[w.id] = (
+                    geo,
+                    w.tags.get("name"),
+                )  # geo is also a string since the format is json, w.id is an int
             except Exception as e:
                 print("error", e)
                 return
 
 
-def geometry_dictionary_to_geojson(geoDict: "dict[str, tuple[str, str]]") -> str:
+def geometry_dictionary_to_geojson(
+    geoDict: dict[str, tuple[str, str]],
```

Comment:
```suggestion
    geoDict: dict[int, tuple[str, str]],
```

Jeg tror det er int det her